src/pages/MoviesPage.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "@emotion/styled";
+import MoviesContainer from "components/movie/MoviesContainer";
+
+const PageContainer = styled.div`
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin: 0 auto;
+  max-width: 1400px;
+  padding: 0.625rem 2.5rem;
+`;
+/**
+ * Represents a page for displaying the whole movie page.
+ */
+function MoviesPage() {
+  return (
+    <PageContainer>
+      <h2 style={{ width: "100%" }}>Popular Movies</h2>
```

Comment:
Instead of using inline styles, please define the h2 tag as a styled component like the pageContainer above. This keeps your code cleaner and makes it easier to manage styles across the application.

Reply:
Also, one suggestion would be to move all the constants in the application to a `constants.js` file and import the data you need.
In this file, it would be the `Popular Movies` to move.